docs/references/concepts.md

```diff
@@ -4,16 +4,35 @@ description: Fundamental user-facing concepts
 
 # Key Concepts
 
+## Instance
+
+An "instance" of Opstrace runs inside your cloud account.
+It is installed with the CLI `opstrace create` command.
+This name must be _globally_ unique and limited to 23 characters (`[a-z0-9-_]`) because it used to provide a conveient DNS name:  `<instance_name>.opstrace.io`.
+Once chosen, an instance name cannot be changed.
+
 ## Tenants
 
-The concept of multitenancy allows for isolating environments within the same Opstrace cluster.
-Tenants are logically isolated, but otherwise share the same \(physical\) base system.
+A "tenant" is a unit of isolation.
+[Multi-tenant systems](https://en.wikipedia.org/wiki/Multitenancy) allow for logical separation of concerns, while otherwise sharing the same underlying system.
+
+Opstrace instances support multiple, secured tenants.
+The tenant primitive extended from the underlying projects—Cortex and Loki.
```

Comment:
Is this sentence correct? "extended" vs "is extended"? Maybe "builds on top of .."?

Is the primary purpose of this sentence to make a shout-out/ an attribution?

Maybe something like 

"The tenant primitive in Opstrace walks on the shoulders of giants; it is built on top of the multi-tenancy capabilities in Cortex and Loki"

?

Reply:
> Is the primary purpose of this sentence to make a shout-out/ an attribution?

I do not think a shout-out is the same thing as attribution.  The purpose of this sentence is to explain where the concept comes from.  In other words, we are clarifying that we did not create it.